pkg/runner/docker/docker.go

```diff
@@ -94,11 +112,24 @@ func (d *docker) Create(ctx context.Context, r *core.RunnerOptions) core.Contain
 }
 
 func (d *docker) Destroy(ctx context.Context, r *core.RunnerOptions) error {
-	if err := d.client.ContainerStop(ctx, r.ContainerID, nil); err != nil {
+	gracefulyContainerStopDuration := time.Second * 10
```

Comment:
make it const

Reply:
making it package level variable as the ContainerStop() expects pointer